.github/workflows/ci.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       with:
         ruby-version: ${{ matrix.ruby }}
         bundler-cache: true
-    - run: ./mutant.sh --since HEAD~1 -- 'Mutant*'
+    - run: bundle exec mutant run --zombie --since HEAD~1
```

Comment:
Should `mutant.sh` be deleted?

Reply:
yes, good catch. Happy to se it go.

I fought myself to add `zombie: true` to the config file loader, and decided against it.

Nobody who is not willing to read mutants source should ever use `--zombie`.

Reply:
> I fought myself to add zombie: true to the config file loader, and decided against it.

That seems like the right call.